service/src/test/kotlin/fi/ouka/evakaoulu/invoice/service/InvoiceSenderTest.kt

```diff
@@ -0,0 +1,34 @@
+// SPDX-FileCopyrightText: 2021 City of Oulu
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.ouka.evakaoulu.invoice.service
+
+import fi.ouka.evakaoulu.EvakaOuluProperties
+import fi.ouka.evakaoulu.IntimeProperties
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.*
+
+internal class InvoiceSenderTest {
+
```

Comment:
Suggestion: When working with clients it´s good to write/have a test for the most common exception -> connection timeout/ not available

Reply:
This is already tested with this 

```
@Test
    fun `should return unsuccessfully sent invoices in failed list`() 
```